tests/common/chain_mock.rs

```diff
@@ -206,14 +207,16 @@ pub async fn spawn_mock_chain(
         .await?
         .await?;
```

Comment:
For some reason changing this call to:
```
identity_manager
     .initialize_v2(delete_verifiers.address())
     .call()
     .await?;
 ```
doesn't work. It compiles but then transactions are being failed. I don't want now to focues on that so just changing other calls.